core-web/libs/sdk/angular/README.md

```diff
@@ -1,229 +1,642 @@
+
+const dotCMSClient: DotCMSClient = createDotCMSClient({
+    dotcmsUrl: 'https://your-dotcms-instance.com',
+    authToken: 'your-auth-token', // Optional for public content
+    siteId: 'your-site-id' // Optional site identifier/name
+});
+
+export const DOTCMS_CLIENT_TOKEN = new InjectionToken<DotCMSClient>('DOTCMS_CLIENT');
 
 export const appConfig: ApplicationConfig = {
     providers: [
-        provideRouter(routes),
         {
             provide: DOTCMS_CLIENT_TOKEN,
-            useValue: DotCmsClient.init(DOTCMS_CLIENT_CONFIG),
+            useValue: dotCMSClient
         }
-    ],
+    ]
 };
 ```
 
-This way, we will have access to `DOTCMS_CLIENT_TOKEN` from anywhere in our application.
+This configuration makes the dotCMS client service available throughout your Angular application, allowing you to inject it wherever needed. For more details on how Angular's InjectionToken works, you can refer to the [Angular InjectionToken documentation](https://angular.dev/api/core/InjectionToken).
+
+### Proxy Configuration for Static Assets
+
+Configure a proxy to leverage the powerful dotCMS image API, allowing you to resize and serve optimized images efficiently. This enhances application performance and improves user experience, making it a strategic enhancement for your project.
 
-### Client Usage
-To interact with the client and obtain information from, for example, our pages
+#### 1. Create a Proxy Configuration
+
+Create a `proxy.conf.json` file in your project:
+
+```json
+// proxy.conf.json
+{
+    "/dA": {
+        "target": "http://localhost:8080", // Your dotCMS instance URL
+        "secure": false, // Set to true if using HTTPS
+        "changeOrigin": true // Required for hosting scenarios
+    }
+}
+```
+
+#### 2. Update Angular Configuration
+
+Add the proxy configuration to your `angular.json`:
+
+```json
+// angular.json
+{
+    "projects": {
+        "my-app": {
+            "architect": {
+                "serve": {
+                    "builder": "@angular-devkit/build-angular:dev-server",
+                    "options": {
+                        "proxyConfig": "src/proxy.conf.json"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+#### 3. Usage in Components
+
+Once configured, image URLs in your components will automatically be proxied to your dotCMS instance:
 
 ```typescript
-export class YourComponent {
-    private readonly client = inject(DOTCMS_CLIENT_TOKEN);
-
-    this.client.page
-        .get({ ...pageParams })
-        .then((response) => {
-            // Use your response
-        })
-        .catch((e) => {
-            const error: PageError = {
-                message: e.message,
-                status: e.status,
-            };
-            // Use the error response
-        })
+// /components/my-dotcms-image.component.ts
+@Component({
+    template: `
+        <img [src]="'/dA/' + contentlet.inode" alt="Asset from dotCMS" />
```

Comment:
Probably we want to add this to our SDK: https://docs.google.com/document/d/1n1O9SMNh2z3_LriSiX8Dd9F4pryfVNqMwjY9azL0vNU/edit?tab=t.0#heading=h.suaidt1do8jz